securitas/templates/user.html

```diff
@@ -1,88 +1,76 @@
-{% extends "with_navbar.html" %}
+{% extends "main.html" %}
 
 {% block title %}User: {{ user.username }}{% endblock %}
 
-{% block head %}
-  {{ super() }}
-  {{ css("user") }}
-{% endblock %}
-
 {% block content %}
   {{ super() }}
-  <div class="container section">
-
+  <div class="container">
     <div class="row">
-      <div class="col s12 l3">
-        <h3 class="header light-blue-text text-darken-4 fw300">
-          {{ user.name if user.name else user.username }}
-        </h3>
-        <div class="card">
-          <div class="card-image">
-            <img src="{{ gravatar(user.mail if user.mail else 'default', 400) }}">
-          </div>
-          <div class="card-stacked">
-            <div class="card-content">
-              <span class="card-title">{{ user.username }}</span>
-              {% if user.username == current_user.username %}
-                <p>
-                  <a href="{{ url_for('user_edit', username=user.username) }}">Edit profile</a>
-                </p>
-              {% endif %}
+      <div class="col-4">
+        <h4 class="" id="user_username">{{ user.username }}</h4>
+        <div id="user_fullname">{{ user.name if user.name else user.username }}</div>
+        <img class="img-thumbnail" src="{{ gravatar(user.mail if user.mail else 'default', 400) }}">
+        {% if user.username == current_user.username %}
+          <a href="{{ url_for('user_edit', username=user.username) }}" class="btn btn-outline-primary btn-block mb-2">Edit profile</a>
```

Comment:
Maybe add some top margin so the borders don't collide with the avatar?

Reply:
done in 
https://github.com/fedora-infra/securitas/pull/92/commits/9660e4ab46895205f6dfee895778531e6a920ca4